src/tokenized-buffer.coffee

```diff
@@ -349,13 +349,23 @@ class TokenizedBuffer extends Model
 
   indentLevelForLine: (line) ->
     if match = line.match(/^[\t ]+/)
-      leadingWhitespace = match[0]
-      tabCount = leadingWhitespace.match(/\t/g)?.length ? 0
-      spaceCount = leadingWhitespace.match(/[ ]/g)?.length ? 0
-      tabCount + (spaceCount / @getTabLength())
```

Comment:
I'm a bit surprised at how this method was counting tabs and spaces before -- just counting the number of spaces and tabs, regardless of where they're located. I think your approach in `parseIndentation` is much better. :+1: